opentreemap/treemap/templates/treemap/partials/udf_edit_field.html

```diff
@@ -9,6 +9,7 @@
 </select>
 {% elif field.type == 'date' %}
 <input class="form-control" type="text" data-field-name="{{ field.name }}"
+       data-date-start-date="{{ datepicker_start_date|date:request.instance.short_date_format|datepicker_format }}"
```

Comment:
I upgraded the bootstrap datepicker and enabled `assumeNearbyYear`. I also verified that the start date option is inclusive, so entering "1/1/1900" will work. I did not add a toast error message.